tests/rustdoc/reexport-attr-merge.rs

```diff
@@ -12,22 +12,19 @@ pub struct Foo;
 
 #[doc(hidden, no_inline, cfg(feature = "bar"))]
 pub use Foo as Foo1;
-
```

Comment:
Please keep the empty lines here and below (easier to read and also smaller diff).